lib/bpf/bpf.h

```diff
@@ -0,0 +1,20 @@
+#ifndef LIB_BPF_BPF_H_
+#define LIB_BPF_BPF_H_
+
+#include <sys/syscall.h>
+
+#include "lib/error/status_or.h"
+#include "lib/posix/syscall.h"
+
+namespace bpf {
+
+// bpf() syscall wrapper. No C standard library support currently exists.
+//
+// See "man 2 bpf" for details.
+inline error::StatusOr<int> Bpf(const int cmd, bpf_attr* const attr) {
```

Comment:
With this, are we moving away from using libbpf for userspace?

Reply:
I think it is too early to say. All I'm doing is experimenting with the kernel interface to better understand what is going on and checking in the utilities I make along the way.

I suspect that we may want to manipulate BPF maps directly, but it is not certain.